docs/source/howto/installation.rst

```diff
@@ -18,7 +18,149 @@ Managing profiles
+
+You can test that everything works by first echoing the environment variable with ``echo $AIIDA_PATH`` to confirm it prints the correct path.
+Finally, you can check that AiiDA know also properly realizes the new location for the configuration folder by calling ``verdi profile list``.
+This should display the current location of the configuration directory:
+
+.. code:: bash
+
+    Info: configuration folder: /home/user/.virtualenvs/aiida/.aiida
+    Critical: configuration file /home/user/.virtualenvs/aiida/.aiida/config.json does not exist
+
+The second line you will only see if you haven't yet setup a profile for this AiiDA instance.
+For information on setting up a profile, refer to :ref:`creating profiles<how-to:installation:profile>`.
+
+Besides a single path, the value of ``AIIDA_PATH`` can also be a colon-separated list of paths.
+AiiDA will go through each of the paths and check whether they contain a configuration directory, i.e., a folder with the name ``.aiida``.
+The first configuration directory that is encountered will be used as the configuration directory.
+If no configuration directory is found, one will be created in the last path that was considered.
+For example, the directory structure in your home folder ``~/`` might look like this::
+
+    .
+    ├── .aiida
+    └── project_a
+        ├── .aiida
+        └── subfolder
+
+If you leave the ``AIIDA_PATH`` variable unset, the default location ``~/.aiida`` will be used.
+However, if you set:
+
+.. code:: bash
+
+    export AIIDA_PATH='~/project_a:'
+
+the configuration directory ``~/project_a/.aiida`` will be used.
+
+.. warning::
+    If there was no ``.aiida`` directory in ``~/project_a``, AiiDA would have created it for you, so make sure to set the ``AIIDA_PATH`` correctly.
+
+
+.. _how-to:installation:configure:tab-completion:
+
+Activating tab-completion
+-------------------------
+The ``verdi`` command line interface has many commands and parameters, which can be tab-completed to simplify its use.
+To enable tab-completion, the following shell command should be executed:
+
+.. code:: bash
+
+    eval "$(_VERDI_COMPLETE=source verdi)"
+
+Note that this *only* enables tab-completion in the *current shell*.
+In order to make this change permanent and have it automatically applied to any shell instance, you can add the command to the start up script.
+This file is shell specific, but likely one of the following:
+
+    * the startup file of your shell (``.bashrc``, ``.zsh``, ...), if aiida is installed system-wide
+    * the `activation script <https://virtualenv.pypa.io/en/latest/userguide/#activate-script>`_ of your virtual environment
+    * a `startup file <https://conda.io/docs/user-guide/tasks/manage-environments.html#saving-environment-variables>`_ for your conda environment
+
+
+.. important::
+
+    After you have added the line to the start up script, make sure to restart the terminal or source the script for the changes to take effect
```

Comment:
Does the `.. important::` directive require two empty lines surrounding it?

Reply:
It doesn't require anything different from the other markup marks such as `.. note::` but I see just a single empty line. Did you mean two linebreaks? I personally like the empty line for readability, but happy to change. What would the standard be? Directly following the mark, or a single linebreak? I guess the former would become problematic if the note contains multiple sentences. So I would vote for either 1 or 2 linebreaks